services/install-mongodb.sh

```diff
@@ -1,9 +1,56 @@
 #!/bin/bash
 
-function install {
+MODE=$3
```

Comment:
What is the purpose of `MODE=$3`?

I think you are trying to get some argument from `services.sh` since you changed the `checkargn` from 2 to 3 but I don't see how a 3rd argument (or any argument for that matter) is passed to the install script from `services.sh`.

Reply:
If you put express after install,  express should be stored in $3.
$1 is mongodb and planet, and $2 is the second argument, such as install and up